debezium-core/src/main/java/io/debezium/relational/RelationalDatabaseConnectorConfig.java

```diff
@@ -283,6 +284,25 @@ public static DecimalHandlingMode parse(String value, String defaultValue) {
             .withDescription("A comma-separated list of regular expressions matching fully-qualified names of columns that should "
                     + "be masked by hashing the input. Using the specified hash algorithms and salt.");
 
+    protected static final ConfigDefinition CONFIG_DEFINITION = CommonConnectorConfig.CONFIG_DEFINITION.edit()
+            .type(
+                    SERVER_NAME)
+            .connector(
+                    DECIMAL_HANDLING_MODE,
+                    TIME_PRECISION_MODE,
+                    SNAPSHOT_LOCK_TIMEOUT_MS)
+            .events(
+                    COLUMN_WHITELIST,
+                    COLUMN_BLACKLIST,
+                    TABLE_WHITELIST,
+                    TABLE_BLACKLIST,
+                    TABLE_IGNORE_BUILTIN,
+                    SCHEMA_WHITELIST,
+                    SCHEMA_BLACKLIST,
```

Comment:
@jpechane, They are in exclude for some others, that still seems inconsistent?

Reply:
Ah, no, exclude is no group (that's really confusing!) but meant to exclude some inherited ones.

Taking a step back, did you re-group any fields, or is the grouping the same?

Reply:
@gunnarmorling Grouping is the same - it is possible that there was incosistent groupings accorss connectors so it was alighned then but I don't recall any intentional regrouping.

Reply:
@gunnarmorling You can rename the method to make the intent clearer.